configDiffer/diff.py

```
@@ -0,0 +1,55 @@
+        if diff is not None:
+            line_diffs[i] = diff
+            end = i
+    if len(file1) != len(file2):
+        end = len(file1) - 1 if len(file1) > len(file2) else len(file2) - 1
+    return ParagraphDiff(begin=start, end=end, line_diffs=line_diffs)
+
+
+def diff_line(line1: str, line2: str) -> LineDiff | None:
+    start = None
+    end = None
+    for i, (l1, l2) in enumerate(zip(line1, line2)):
```

Ruff (0.11.9)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)